scripts/chime8/installers/install_c8_dependencies.sh

```diff
@@ -0,0 +1,5 @@
+pip install optuna
+pip install git+https://github.com/chimechallenge/chime-utils
+pip install git+https://github.com/kpu/kenlm
+pip install cmake>=3.18
+sh ./installers/run_install_lm.sh ${PWD}/../../../C8DASR-Baseline-NeMo
```

Comment:
should remove the `./installers` prefix from the path?

Reply:
yes good catch